src/lib.rs

```diff
@@ -219,12 +239,14 @@ mod tests {
         }
 
         impl super::OutputPin for &'_ mut TestPin {
-            fn set_high(&mut self) {
-                (*self).0 = 1;
+            type Error = ();
```

Comment:
Cool, didn't knew about this

Reply:
This requires `std`. I guess `core::convert::Infallible` works as well?

Reply:
Yeah, sorry, I meant `core` it should be there as well, since it's used for some integer conversions, if I remember correctly (`u8` -> `usize` for instance).